lib/src/visitor/async_evaluate.dart

```diff
@@ -525,11 +525,9 @@ class _EvaluateVisitor
       {Uri baseUrl,
       Map<String, _ConfiguredValue> configuration,
       bool namesInErrors = false}) async {
-    configuration ??= const {};
-
     var builtInModule = _builtInModules[url];
     if (builtInModule != null) {
-      if (configuration.isNotEmpty) {
+      if (configuration != null && configuration.isNotEmpty) {
```

Comment:
Would the better style here and in line 661 be `configuration?.isNotEmpty ?? false`?

Reply:
I considered that, but I think the null logic is easier to follow here than trying to follow a bunch of `?`s. It's close, though.